spec/intl.html

```diff
@@ -52,6 +52,45 @@ <h1>Time Zone Identifiers</h1>
     </p>
   </emu-clause>
 
+  <emu-clause id="sup-availablenamedtimezoneidentifiers" oldids="sec-availabletimezones" type="implementation-defined abstract operation">
+    <h1>AvailableNamedTimeZoneIdentifiers ( ): a List of Time Zone Identifier Records</h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>
+        Its result describes all available named time zone identifiers in this implementation, as well as the primary time zone identifier corresponding to each available named time zone identifier.
+        The List is sorted by [[Identifier]] of each Time Zone Identifier Record.
+      </dd>
+      <dt>redefinition</dt>
+      <dd>true</dd>
+    </dl>
+    <emu-alg>
+      1. Let _identifiers_ be a List containing the String value of each Zone or Link name in the IANA Time Zone Database that is supported by the implementation.
+      1. Assert: No element of _identifiers_ is an ASCII-case-insensitive match for any other element.
+      1. Assert: Every element of _identifiers_ identifies a Zone or Link name in the IANA Time Zone Database.
+      1. <ins>If no element of _identifiers_ is an ASCII-case-insensitive match for *"Etc/Unknown"*, append *"Etc/Unknown"* to _identifiers_.</ins>
+      1. Set _identifiers_ to SortStringListByCodeUnit(_identifiers_).
+      1. Let _result_ be a new empty List.
+      1. For each element _identifier_ of _identifiers_, do
+        1. Let _primary_ be _identifier_.
+        1. If _identifier_ is a non-primary time zone identifier in this implementation and _identifier_ is <del>not *"UTC"*</del><ins>neither *"UTC"* nor *"Etc/Unknown"*</ins>, then
```

Comment:
I was concerned that this would add complexity because the implementation would have to insert it in the right place in the sorted list, instead of just being able to append one element like could be done before the sorting step. What do you think?

Reply:
Ah, I see. I didn't spot that `identifiers` is sorted before the loop.